src/pages/PostJobs/validate.tsx

```diff
@@ -0,0 +1,39 @@
+import  {JobDet}  from '../../components/DataModels/JobDet';
+function isDateEmpty(date: Date): boolean {
+  return !date || isNaN(date.valueOf());
+}
+const Validate=(jobView:JobDet)=>{
+  const {title,qualification,experience}= jobView.job;
+  const {name,type,logo}= jobView.company;
+  const {city,state,country,region}= jobView.location;
+  const {postingDate,expiryDate,closingDate,removingJobDate}= jobView.dates;
+  const salary=jobView.salary;
+  const qualifications =jobView.qualifications;
+  const duties= jobView.duties;
+
+
+  if(title.trim()==='' || qualification.trim()==='' || experience.trim()===''){
```

Comment:
you can just replace this `title.trim() === ''` to `!title`. that should be fine mostly.